integration-tests/cypress.config.js

```diff
@@ -11,7 +17,11 @@ module.exports = defineConfig({
     configFile: 'reporter-config.json',
   },
   fixturesFolder: 'fixtures',
-  defaultCommandTimeout: 5000,
+  chromeWebSecurity: false,
+  env: {
+    TAGS: '@pipelines and (@pre-condition or @smoke or @regression) and not (@manual or @to-do or @un-verified or @broken-test)',
```

Comment:
I understand why the tag `@pipelines` was needed in the past but do we still need it?